sql/analyze_test.go

```diff
@@ -208,6 +209,9 @@ func TestSimplifyExpr(t *testing.T) {
 		{`(a, b) = (1, 2)`, `(a, b) IN ((1, 2))`, true},
 		{`(a, b) = (1, 2) OR (a, b) = (3, 4)`, `(a, b) IN ((1, 2), (3, 4))`, true},
 		{`(a, b) IN ((2, 1), (1, 2), (1, 2), (2, 1))`, `(a, b) IN ((1, 2), (2, 1))`, true},
+		// Expressions that don't simplify properly:
```

Comment:
I assume this really means that the expressions don't simplify when they could. That is, the current behavior isn't erroneous, just not optimal.